test/server/options_impl_test.cc

```diff
@@ -115,6 +122,9 @@ TEST_F(OptionsImplTest, All) {
   EXPECT_EQ(5U, options->baseId());
   EXPECT_TRUE(options->useDynamicBaseId());
   EXPECT_EQ("/foo/baz", options->baseIdPath());
+  EXPECT_EQ("/foo/envoy_domain_socket", options->socketPath());
+  // Decimal value of 644 is 420
+  EXPECT_EQ(420, options->socketMode());
```

Comment:
I think you can use `0644`, which C will interpret as octal (because of leading 0), instead of `420`

Reply:
Done